src/frontend/app/toolbar/import-file/import-file.component.html

```diff
@@ -2,5 +2,11 @@
        id="file"
        [accept]="supportedFileTypes"
        (change)="handleFileInput($event.target?.files)"
-       [formControl]="inputFile">
-<label for="file">Import file</label>
\ No newline at end of file
+       [formControl]="inputFile"
+       class="import-file__input">
+<div class="import-file__label-container">
```

Comment:
Could we use a mat-button instead of the div? Or at least the ripple effect.

Reply:
Done!